src/App.js

```diff
@@ -1,43 +1,53 @@
-import "./App.css";
-import Tweet from "./components/Tweet";
+import './App.css';
+import Tweet from './components/Tweet';
 
 const tweetsArray = [
   {
     user: {
-      name: "Thoughts of Dog®",
-      image: "https://i.imgur.com/b0EdHVV.jpg",
-      handle: "dog_feelings",
+      name: 'Thoughts of Dog®',
+      image: 'https://i.imgur.com/b0EdHVV.jpg',
+      handle: 'dog_feelings',
     },
-    timestamp: "1h ago",
+    timestamp: '1h ago',
     message:
-      "the human likes to say. that i live here rent free. but i would argue. this housing accommodation. is my payment. for a lifetime of love. and excellent company",
+      'the human likes to say. that i live here rent free. but i would argue. this housing accommodation. is my payment. for a lifetime of love. and excellent company',
   },
   {
     user: {
-      name: "Thoughts of Dog®",
-      image: "https://i.imgur.com/b0EdHVV.jpg",
-      handle: "dog_feelings",
+      name: 'Thoughts of Dog®',
+      image: 'https://i.imgur.com/b0EdHVV.jpg',
+      handle: 'dog_feelings',
     },
-    timestamp: "2h ago",
+    timestamp: '2h ago',
     message:
-      "sometimes. the human presses their noggin against mine. to figure out what i’m thinking. so i just think really hard. about how much i love them. and hope they figure it out",
+      'sometimes. the human presses their noggin against mine. to figure out what i’m thinking. so i just think really hard. about how much i love them. and hope they figure it out',
   },
   {
     user: {
-      name: "Thoughts of Dog®",
-      image: "https://i.imgur.com/b0EdHVV.jpg",
-      handle: "dog_feelings",
+      name: 'Thoughts of Dog®',
+      image: 'https://i.imgur.com/b0EdHVV.jpg',
+      handle: 'dog_feelings',
     },
-    timestamp: "3h ago",
+    timestamp: '3h ago',
     message:
-      "here is what. i plan to accomplish today: \n\n2. bark loudly. but at nothing \n7. lose my ball under the couch\n7b. politely ask the human. to get my ball\n3. immediately lose it again. under the same couch\n4. big nap. you have worked hard\n2. repeat",
+      'here is what. i plan to accomplish today: \n\n2. bark loudly. but at nothing \n7. lose my ball under the couch\n7b. politely ask the human. to get my ball\n3. immediately lose it again. under the same couch\n4. big nap. you have worked hard\n2. repeat',
   },
 ];
 
+function displayAll() {
+  for (let i = 0; i < tweetsArray.length; i++) {
+    return i;
+  }
+}
```

Comment:
Here, this function is not doing anything. If you want to display the same component multiple times you should use map like you did down below